.github/PULL_REQUEST_TEMPLATE/develop-template.md

```diff
@@ -0,0 +1,16 @@
+Originator(s):
+
+Summary (include the keyword ['closes', 'fixes', 'resolves'] and issue number):
+
```

Comment:
Perhaps add a bullet like:  `Purpose of PR (new scheme, modification to existing scheme, bug fix to scheme, etc.)`.  This might be unnecessary if the PR title covers this, but if it doesn't, perhaps this information would be useful?

Reply:
For this section (summary and purpose), would it make sense to follow a layout similar to a few other open sourced projects (ex. https://github.com/dotnet/aspnetcore/pull/55968), where we have `Summary`, `Description` and then a section for fixes/resolutions, followed by the other sections below?